hack/jenkins/job-configs/kubernetes-jenkins/kubernetes-e2e.yaml

```diff
@@ -756,48 +756,78 @@
     jobs:
         - 'kubernetes-e2e-gce-trusty-{suffix}'
 
-# Jobs that run e2e tests on GKE with Trusty as node image on the release-1.1 branch.
+# Jobs that run e2e tests on GKE with Trusty as node image on the release-1.2 branch.
 - project:
     name: kubernetes-e2e-gke-trusty
-    trigger-job: 'kubernetes-build-1.1'
+    trigger-job: 'kubernetes-build-1.2'
     test-owner: 'wonderfly@google.com'
-    branch: 'release-1.1'
-    runner: '{old-runner-1-1}'
-    post-env: ''
+    provider-env: |
+        {gke-provider-env}
     emails: 'wonderfly@google.com,qzheng@google.com'
     suffix:
+        - 'gke-trusty-test':
+            description: 'Run E2E tests on GKE test endpoint.'
+            timeout: 480
+            job-env: |
+                export PROJECT="kubekins-e2e-gke-trusty-test"
+                export CLOUDSDK_BUCKET="gs://cloud-sdk-testing/rc"
+                export JENKINS_USE_SERVER_VERSION="y"
+                export E2E_NAME="jkns-gke-e2e-test-trusty"
+        - 'gke-trusty-subnet':
+            description: 'Run E2E tests on GKE test endpoint in a subnet.'
+            timeout: 480
+            job-env: |
+                # auto-subnet manually created - if deleted, it will need to be recreated
+                # gcloud alpha compute networks create auto-subnet --mode auto
+                export E2E_NAME="auto-subnet"
+                export PROJECT="k8s-e2e-gke-trusty-subnet"
+                export CLOUDSDK_BUCKET="gs://cloud-sdk-testing/rc"
+                export JENKINS_USE_SERVER_VERSION="y"
+                export E2E_NAME="jkns-gke-e2e-subnet-trusty"
+        - 'gke-trusty-staging':
+            description: 'Run E2E tests on GKE staging endpoint.'
+            timeout: 480
+            job-env: |
+                export PROJECT="e2e-gke-trusty-staging"
+                export CLOUDSDK_BUCKET="gs://cloud-sdk-testing/rc"
+                export JENKINS_USE_SERVER_VERSION="y"
+                export CLOUDSDK_API_ENDPOINT_OVERRIDES_CONTAINER="https://staging-container.sandbox.googleapis.com/"
+                export E2E_NAME="jkns-gke-e2e-staging-trusty"
+        - 'gke-trusty-staging-parallel':
+            description: 'Run E2E tests on GKE staging endpoint in parallel.'
+            timeout: 80
+            job-env: |
+                export PROJECT="e2e-gke-trusty-staging-p"
+                export CLOUDSDK_BUCKET="gs://cloud-sdk-testing/rc"
+                export JENKINS_USE_SERVER_VERSION="y"
+                export CLOUDSDK_API_ENDPOINT_OVERRIDES_CONTAINER="https://staging-container.sandbox.googleapis.com/"
+                export GINKGO_TEST_ARGS="--ginkgo.skip=\[Slow\]|\[Serial\]|\[Disruptive\]|\[Flaky\]|\[Feature:.+\]"
```

Comment:
1.2 is pushed out to staging & prod, so we need to update our configs (here and above) to reflect that.  I will send a PR shortly & cc you.